content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1035,13 +1035,13 @@ by the cloud-controller-manager.
 
 ### batch.kubernetes.io/job-completion-index
 
-Type: Annotation
+Type: Label and Annotation
```

Comment:
Also add an entry for the new statefulset label

Reply:
ah of course - added!